sessionctx/variable/sysvar.go

```diff
@@ -706,6 +706,7 @@ var defaultSysVars = []*SysVar{
 	{ScopeGlobal | ScopeSession, TiDBSkipIsolationLevelCheck, BoolToIntStr(DefTiDBSkipIsolationLevelCheck)},
 	/* The following variable is defined as session scope but is actually server scope. */
 	{ScopeSession, TiDBGeneralLog, strconv.Itoa(DefTiDBGeneralLog)},
+	{ScopeSession, TiDBPProfSQLCPU, strconv.Itoa(DefTiDBPProfSQLCPU)},
```

Comment:
As https://github.com/pingcap/tidb/pull/14312/files#diff-dc417f5322100e2e83a8818c70a25878R810, Is it really in session scope?

Reply:
It is another issue.

Reply:
Could we set `util.EnablePProfSQLCPU` into `session.sessionVars`?

Reply:
there are no benefit to attach a server scope var to per session, but we can move it into variable package...moved

Reply:
> As https://github.com/pingcap/tidb/pull/14312/files#diff-dc417f5322100e2e83a8818c70a25878R810, Is it really in session scope?

>	/* The following variable is defined as session scope but is actually server scope. */

the comments above those variable talk some about this and it seems infra-team have plan to make a real server scope variable in roadmap